.github/workflows/unit-tests.yml

```
@@ -0,0 +1,24 @@
+name: Unit Tests
+on:
+  pull_request:
+    branches:
+      - dev
+jobs:
+  unit_tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)